interface-spec.html

```diff
@@ -169,12 +181,14 @@ <h3><dfn>NamedNode</dfn> interface</h3>
       <dfn>value</dfn> the IRI of the named node (example: <code>"http://example.org/resource"</code>).
     </p>
     <p>
-      <dfn>equals()</dfn> returns <code>true</code> if and only if <code>other</code> has
-      <code>termType</code> <code>"NamedNode"</code> and the same <code></code>value</code>.
+      <dfn>equals()</dfn> returns <code>true</code> if and only if
+      <a href="#term-equals">all general <code>Term#equals</code> conditions</a> hold
```

Comment:
Respec would also handle this case if `<a>Term.equals</a>` is used. As the link would not cover the current text, maybe change it to:

`if and only if all general condititions defined in <a>Term.equals</a>`

Reply:
Mmm, don't personally like this, but WebIDL has to be followed, thanks for noticing.